api/api_public_transaction_pool.go

```diff
@@ -596,8 +610,11 @@ func (s *PublicTransactionPoolAPI) RecoverFromMessage(
 	}
 
 	key := state.GetKey(address)
-	if key.ValidateMember(pubkey, address) {
-		return crypto.PubkeyToAddress(*pubkey), nil
+	if key.ValidateMember(klayRpk, address) {
+		return crypto.PubkeyToAddress(*klayRpk), nil
+	}
+	if key.ValidateMember(ethRpk, address) {
+		return crypto.PubkeyToAddress(*ethRpk), nil
```

Comment:
I do agree that `klayEcRecover` and `ethEcRecover` realistically won't fail. But just in case only one of them succeeds,

```suggestion
	var recoverErr error
	if pubkey, err := klayEcRecover(data, sig); err == nil {
		if key.ValidateMember(pubkey, address) {
			return crypto.PubkeyToAddress(*pubkey), nil
		}
	} else {
		recoverErr = err
	}
	if pubkey, err := ethEcRecover(data, sig); err == nil {
		if key.ValidateMember(pubkey, address) {
			return crypto.PubkeyToAddress(*pubkey), nil
		}
	} else {
		recoverErr = err
	}

	if recoverErr != nil {
		return common.Address{}, recoverErr
	} else {
		return common.Address{}, errors.New("Invalid signature")
	}
```